server/notebooks/api_views.py

```diff
@@ -147,3 +148,4 @@ def perform_create(self, serializer):
                 )
 
         serializer.save(**ctx)
+        execute_notebook_revisions_cleanup(ctx["notebook_id"])
```

Comment:
This should use the schedule API and not call the function itself otherwise it'll just be called in-process, e.g.:
```python
        tasks.schedule(execute_notebook_revisions_cleanup, ctx["notebook_id"])        
```
Here's an example:
https://github.com/iodide-project/iodide/blob/1891a5995e0e8a6bd318026e2988a207d7b71641/server/files/api_views.py#L150

Reply:
Thanks